administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE `#__assets` SET `title` = 'PUBLISH' WHERE `title`= 'Publish';
+UPDATE `#__assets` SET `title` = 'UNPUBLISH' WHERE `title`= 'Unpublish';
+UPDATE `#__assets` SET `title` = 'TRASH' WHERE `title`= 'Trash';
+UPDATE `#__assets` SET `title` = 'ARCHIVE' WHERE `title`= 'Archive';
+UPDATE `#__assets` SET `title` = 'FEATURE' WHERE `title`= 'Feature';
+UPDATE `#__assets` SET `title` = 'UNFEATURE' WHERE `title`= 'Unfeature';
+UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `title`= 'Publish & Feature';

```

Comment:
Maybe we should also add the asset name to the where condition as the titles might also be used by 3rd party extensions and we should not update these.

Reply:
Please help me make this right by changing the line here

Reply:
```suggestion
UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Publish & Feature';
```
Do the same for all other update statements here and in the PostgreSQL file (for PostgreSQL use of course other names quotes `"`)
The additional condition added with my suggestion can be the same for all these update statements.

Reply:
Hmm, I just see the other, old PR #35457 for 4.0.3 did not do that either. But I suggest to do it here so we are safer.

Reply:
Thank you, I made the changes